umbraco/Umbraco.CMS.Backoffice
# Chore: Update's VS Code new `UmbLitElement` code snippet

## Description

Updates the VS Code code snippet for creating new `UmbLitElement` classes.

## Types of changes


## Motivation and context

The original contents doesn't follow our current practices.

Changed the command to be `new umb element`, (as the previous `lit new` felt too generalised).

## How to test?

Create a new empty TypeScript file, start typing `new umb element` (which should appear in the autocomplete), press tab/enter to complete... and the class scaffold should appear, with the relevant naming of the class and element names.
